docs/handbook/getting-started.md

```diff
@@ -404,6 +343,9 @@ For example: `cdrecord dev=4,1,0 imagefile.iso`
 
 </div>
 
+* For illumos/Solaris, you may use either `rmformat -l` or `cdrecord --scanbus` to locate your device.
```

Comment:
Fixed....3 substitutions on 3 lines


On 09/05/2016 10:12 AM, Alexander Pyhalov wrote:
>
> In docs/handbook/getting-started.md 
> <https://github.com/OpenIndiana/oi-docs/pull/50#discussion_r77527679>:
>
> > @@ -404,6 +343,9 @@ For example: `cdrecord dev=4,1,0 imagefile.iso`
> >
> >  </div>
> >
> > +* For illumos/Solaris, you may use either `rmformat -l` or `cdrecord --scanbus` to locate your device.
>
> again -scanbus
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/OpenIndiana/oi-docs/pull/50/files/a637c449c62d250af98710d7af3a4dfb8b8d46c9#r77527679>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AP9jzDqBArO4kgMPSGdKIhIkkybjy4Q2ks5qnCNGgaJpZM4J1FOY>.
>